README.md

```diff
@@ -20,12 +20,39 @@ Check [docs/build-n-test.md](docs/build-n-test.md) for details.
 Check [docs/naming-convention.md](docs/naming-convention.md) for details.
 
 ## Performance
-After integrating dcurl into IRI, performance of [attachToTangle](https://iota.readme.io/reference#attachtotangle) is measured as following.
-* Each sampling is measured with 30 transaction trytes and total 200 samples are measured.
-* mwm = 14, 26 CPU threads to find nonce
-* Settings: enable 2 pow tasks in CPU, 1 pow tasks in GPU at the same time
+After integrating dcurl into IRI, performance of [attachToTangle](https://docs.iota.org/docs/node-software/0.1/iri/references/api-reference#attachtotangle) is measured as the following.
+
+* Setting: MWM = 14, 200 attachToTangle API requests with each containing 2 transactions
+* Local CPU:
+  * AMD Ryzen Threadripper 2990WX 32-Core Processor 
+  * 2 PoW tasks at the same time
+  * Each task uses 32 CPU threads to find nonce
+  * SIMD enabled
+* Remote worker:
+  * [DE10-Nano](https://www.terasic.com.tw/cgi-bin/page/archive.pl?Language=English&No=1046) board
```

Comment:
DE10-Nano is a just a reference board. In fact, any hardware equipped with Intel/Altera Cyclone V SoC should perform as expected.

Reply:
Replace with `The board with Intel/Altera Cyclone V SoC`.